installer_scripts/install-yb-voyager

```diff
@@ -832,6 +835,25 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	script_dir_path="guardrail_scripts"
```

Comment:
don't seem like you're using this variable? 
Either way, keep the directory name same in /opt/yb-voyager/x and yb-voyager/x. Would be good to be consistent.

Reply:
Done